src/main/kotlin/com/back/domain/admin/service/AdminService.kt

```
@@ -0,0 +1,149 @@
+    // 회원 정보 수정
+    @Transactional
+    fun updateMemberInfo(memberId: Long, request: AdminUpdateMemberRequest) {
+        log.info("회원 정보 수정 요청 - memberId: {}", memberId)
+
+        val member = findMemberById(memberId)
+
+        try {
+            member.updateName(request.name)
+            member.changeStatus(request.status)
+            updateProfileUrlIfPresent(member, request.profileUrl)
+        } catch (e: IllegalStateException) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)